src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  width: 6rem;
+  display: flex;
+  justify-content: space-between;
+  cursor: pointer;
```

Comment:
Why is the cursor pointer here? Users can't do anything with those stars at this moment, don't confuse them ;)